modules/core/admin.py

```diff
@@ -16,3 +19,75 @@ class AdminCommands(commands.Cog):
     async def load(self, ctx, module: str):
         ctx.bot.load_module(module)
         await ctx.send(f"Successfully loaded {module}!")
+
+    @commands.is_owner(usage='<code>')
+    @commands.command()
+    async def eval(self, ctx):
+        if self._eval.get('env') is None:
+            self._eval['env'] = {}
+        if self._eval.get('count') is None:
+            self._eval['count'] = 0
+
+        self._eval['env'].update({
+            'lookups': self.lookups,
```

Comment:
what is `self.lookups` supposed to refer to?

Reply:
That was depreciated, i forgot to remove it.